core/capture/gif.cpp

```diff
@@ -57,9 +57,9 @@ void gif_new_frame()
     RGB24 *ptr24 = buffer.data();
     for(unsigned int i = 0; i < 320*240; ++i)
     {
-        ptr24->r = (*ptr16 & 0b1111100000000000) >> 8;
-        ptr24->g = (*ptr16 & 0b0000011111100000) >> 3;
-        ptr24->b = (*ptr16 & 0b0000000000011111) << 3;
+        ptr24->r = (*ptr16 & 0xF800) >> 8;
```

Comment:
Qt Designer doesn't let you detect 2015 for automatic setup. I could try to fight with it to get it to work, but that may add a lot of complexity for building on windows.